src/python/pants/backend/python/util_rules/pex_cli.py

```diff
@@ -35,8 +35,8 @@ class PexCli(TemplatedExternalTool):
     name = "pex"
     help = "The PEX (Python EXecutable) tool (https://github.com/pantsbuild/pex)."
 
-    default_version = "v2.1.148"
-    default_url_template = "https://github.com/pantsbuild/pex/releases/download/{version}/pex"
+    default_version = "v2.2.1"
+    default_url_template = "https://github.com/pex-tool/pex/releases/download/{version}/pex"
     version_constraints = ">=2.1.135,<3.0"
```

Comment:
NB. this leaves this at the same constraint it was in 2.19, which is different to what it was in #20587. In particular, #20481 and #20598 aren't cherry-picked.